src/components/gridList/demoDynamicTiles/index.html

```diff
@@ -1,18 +1,17 @@
 <div ng-app="gridListDemo" ng-controller="gridListDemoCtrl as vm" flex>
+  <md-button ng-click="vm.rotate()">Rotate</md-button>
```

Comment:
Might be easier for me to ditch the button. It was useful for testing, but I'm not sure it offers much to the demo.